### PR TITLE
Add Apple Ingot

### PR DESCRIPTION
**Pull request by MetalTurtle18**
This PR adds the apple ingot, which can be crafted with the inverse of the golden apple recipe. It doesn't do anything.

I also reformatted a lot of the code and tried to make it more consistent. *I don't think I broke anything.*

### CODE REVIEW

**Comment by MetalTurtle18** [comment]
I have no idea how the datagen stuff works but I noticed lots of the items didn't have a name in-game.

File: src/main/java/io/github/communitymod/common/items/MiguelItem.java [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
+                } else {
+                    context.getPlayer().spawnAtLocation(Items.DIRT);
+                }
+                itemCoolDowns.addCooldown(stack.getItem(), 200);
+            }
+
+        }
+
+        return InteractionResult.SUCCESS;
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/common/items/MiguelItem.java:40]: Undo formatting.

File: src/main/java/io/github/communitymod/common/items/ModSpawnEggItem.java [hunk 2]
```diff
@@ -15,35 +8,39 @@
...
+            EGGS.put(spawnEgg.getType(null), spawnEgg);
+            DispenserBlock.registerBehavior(spawnEgg, spawnEgg.eggData.getDispenseBehaviour());
+        }
+        UNADDED_EGGS.clear();
+    }
+
+    @Override
+    public EntityType<?> getType(CompoundTag nbt) {
+        return this.entityTypeSupplier.get();
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/common/items/ModSpawnEggItem.java:45]: Undo formatting. Static members at the top.

File: src/main/java/io/github/communitymod/core/CommonEvents.java [hunk 3]
```diff
@@ -17,40 +16,40 @@
...
-		}
-	}
+            }
+        }
+
+        @SubscribeEvent
+        public static void entityAttributes(final EntityAttributeCreationEvent event) {
+            event.put(EntityInit.BEAN_ENTITY.get(), BeanEntity.createAttributes().build());
+        }
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/CommonEvents.java:54]: Undo formatting.

File: src/main/java/io/github/communitymod/core/init/BlockEntityInit.java [hunk 4]
```diff
@@ -7,6 +7,6 @@
 
 public final class BlockEntityInit {
 
-	public static final DeferredRegister<BlockEntityType<?>> BLOCK_ENTITIES = DeferredRegister
-			.create(ForgeRegistries.BLOCK_ENTITIES, CommunityMod.MODID);
+    public static final DeferredRegister<BlockEntityType<?>> BLOCK_ENTITIES = DeferredRegister
+            .create(ForgeRegistries.BLOCK_ENTITIES, CommunityMod.MODID);
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/BlockEntityInit.java:11]: Undo formatting.

File: src/main/java/io/github/communitymod/core/init/BlockInit.java [hunk 5]
```diff
@@ -11,18 +11,18 @@
...
+    public static <T extends Block> void blacklistBlockItem(RegistryObject<T> block) {
+        ItemInit.BLOCK_ITEM_BLACKLIST.add((RegistryObject<Block>) block);
+    }
 
-	public static void toBlacklist() {
-		// blacklistBlockItem(BlockInit.MY_BLOCK);
-	}
+    public static void toBlacklist() {
+        // blacklistBlockItem(BlockInit.MY_BLOCK);
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/BlockInit.java:27]: Undo formatting.

File: src/main/java/io/github/communitymod/core/init/EntityInit.java [hunk 6]
```diff
@@ -13,15 +13,15 @@
...
+                    .build(new ResourceLocation(CommunityMod.MODID, "bean").toString()));
 
-	public static void registerSpawnEggs() {
-		ItemInit.ITEMS.register("bean_spawn_egg",
-				() -> new ModSpawnEggItem(BEAN_ENTITY, new SpawnEggData(0xFFCC88, 0x38FA71)));
-	}
+    public static void registerSpawnEggs() {
+        ItemInit.ITEMS.register("bean_spawn_egg",
+                () -> new ModSpawnEggItem(BEAN_ENTITY, new SpawnEggData(0xFFCC88, 0x38FA71)));
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/EntityInit.java:26]: Undo formatting.

File: src/main/java/io/github/communitymod/core/init/ItemInit.java [hunk 7]
```diff
@@ -31,64 +26,93 @@
...
+    @SubscribeEvent
+    public static void registerBlockItems(final RegistryEvent.Register<Item> event) {
+        final IForgeRegistry<Item> registry = event.getRegistry();
+        BlockInit.BLOCKS.getEntries().stream().filter(object -> !BLOCK_ITEM_BLACKLIST.contains(object))
+                .forEach(block -> {
+                    final var blockItem = new BlockItem(block.get(), new Item.Properties().tab(CommunityMod.TAB));
+                    blockItem.setRegistryName(block.getId());
+                    registry.register(blockItem);
+                });
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/ItemInit.java:117]: Undo formatting.

File: src/main/java/io/github/communitymod/core/init/SoundsInit.java [hunk 8]
```diff
@@ -11,18 +8,21 @@
...
 
-	public static final RegistryObject<SoundEvent> MUSIC_DISC_BEANAL = SOUNDS.register("music.record.beanal",
-			() -> new SoundEvent(new ModResourceLocation("music.record.beanal")));
+    static final Set<RegistryObject<Block>> BLOCK_ITEM_BLACKLIST = new HashSet<>();
 
-	public static void registerSounds() {
+    public static void registerSounds() {
 
-	}
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/SoundsInit.java:26]: Undo formatting.

File: src/main/java/io/github/communitymod/core/util/SpawnEggData.java [hunk 9]
```diff
@@ -12,36 +10,38 @@
...
+    }
 
-	public SpawnEggData setDispenseBehaviour(final DispenseItemBehavior dispenseBehaviour) {
-		this.dispenseBehaviour = dispenseBehaviour;
-		return this;
-	}
+    public SpawnEggData setDispenseBehaviour(final DispenseItemBehavior dispenseBehaviour) {
+        this.dispenseBehaviour = dispenseBehaviour;
+        return this;
+    }
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/util/SpawnEggData.java:46]: Undo formatting.

**Review by DaRealTurtyWurty — CHANGES_REQUESTED** [review]

**Pull request closed without merging**